### ISSUE TITLE
 Expected the `options.agent` properties to be `http`, `https` or `http2`, got `_events`

### ISSUE THREAD

**Issue by justin-calleja** [jdalrymple/gitbeaker#710]
**Description**

**Steps to reproduce**

`npm i -g @gitbeaker/cli`

```
# export GITLAB_TOKEN and GITLAB_HOST env vars
$ gitbeaker projects all
```

**Expected behaviour**

The cli should show some meaningful output related to gitlab projects.

**Actual behaviour**

I get this error (note I have `gb-token` and `gb-host` set as env vars and cli outputs them when I use `-g`):

```
gitbeaker projects all
⠋ Calling GitlabTypeError: Expected the `options.agent` properties to be `http`, `https` or `http2`, got `_events`
    at normalizeArguments (/Users/juscal/.nvm/versions/node/v12.14.1/lib/node_modules/@gitbeaker/cli/node_modules/got/dist/source/core/index.js:502:27)
    at normalizeArguments (/Users/juscal/.nvm/versions/node/v12.14.1/lib/node_modules/@gitbeaker/cli/node_modules/got/dist/source/as-promise/core.js:36:31)
    at got (/Users/juscal/.nvm/versions/node/v12.14.1/lib/node_modules/@gitbeaker/cli/node_modules/got/dist/source/create.js:87:39)
    at /Users/juscal/.nvm/versions/node/v12.14.1/lib/node_modules/@gitbeaker/cli/node_modules/@gitbeaker/node/dist/index.js:1:2178
    at /Users/juscal/.nvm/versions/node/v12.14.1/lib/node_modules/@gitbeaker/cli/node_modules/@gitbeaker/node/dist/index.js:1:1669
    at Object.next (/Users/juscal/.nvm/versions/node/v12.14.1/lib/node_modules/@gitbeaker/cli/node_modules/@gitbeaker/node/dist/index.js:1:1774)
    at /Users/juscal/.nvm/versions/node/v12.14.1/lib/node_modules/@gitbeaker/cli/node_modules/@gitbeaker/node/dist/index.js:1:711
    at new Promise (<anonymous>)
    at i (/Users/juscal/.nvm/versions/node/v12.14.1/lib/node_modules/@gitbeaker/cli/node_modules/@gitbeaker/node/dist/index.js:1:456)
    at /Users/juscal/.nvm/versions/node/v12.14.1/lib/node_modules/@gitbeaker/cli/node_modules/@gitbeaker/node/dist/index.js:1:2049
```

**Comment by jdalrymple** [comment]
> The cli should show some meaningful output related to gitlab projects.
:joy: yes that would be a fair assumption. Ill give it a look! havent seen that error before :/

**Comment by inkidu** [comment]
I don't know if this is related, but when testing 18.1.0 this morning I had to install http2 in order to get my code to compile: @gitbeaker/node -> got -> http2-wrapper. http2-wrapper depends on http2, but node did not install it automatically. Once I installed it manually, things moved forward.

**Comment by jdalrymple** [comment]
> I don't know if this is related, but when testing 18.1.0 this morning I had to install http2 in order to get my code to compile: @gitbeaker/node -> got -> http2-wrapper. http2-wrapper depends on http2, but node did not install it automatically. Once I installed it manually, things moved forward.

Thats odd, but noted, ill see what comes up when the tests run

**Comment by jdalrymple** [comment]
Found the problem!

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`18.2.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/18.2.0) :rocket:
